docs/docs/cmd/spo/site/site-recyclebinitem-restore.md

```diff
@@ -0,0 +1,27 @@
+# spo site recyclebinitem restore
+
+Restores given items from the site recycle bin
+
+## Usage
+
+```sh
+m365 spo site recyclebinitem restore [options]
+```
+
+## Options
+
+`-u, --siteUrl <siteUrl>`
+: URL of the site for which to restore the recycle bin items
+
+`-i, --id <id>`
```

Comment:
In all other commands we expect only a single ID in the `id` option. Since we're allowing multiple IDs here, we should rename the option to `ids` to make that explicit.

Reply:
same question and comment here. @garrytrinder  what do you think ?

Reply:
changed